cmd/operator-sdk/bundle/validate.go

```diff
@@ -150,27 +134,61 @@ func (c bundleValidateCmd) validate(args []string) error {
 // exit code to be returned (true by default).
 func (c *bundleValidateCmd) addToFlagSet(fs *pflag.FlagSet) {
 	fs.StringVarP(&c.imageBuilder, "image-builder", "b", "docker",
-		"Tool to extract bundle image data. Only used when validating a bundle image. "+
-			"One of: [docker, podman]")
+		"Tool to pull and unpack bundle images. Only used when validating a bundle image. "+
+			"One of: [docker, podman, none]")
```

Comment:
IMO we need to make clear that it is not a valid option when the bundle was created with the flag `--generate-only
`. We know that when this flag is used an image is not created and in this way, it is not valid for`Only used when validating a bundle image`. 

However, we can make the info more implicit and directly instead of assuming that they will come this conclusion easily.   

```suggestion
		"Tool to pull and unpack bundle images. Only used when validating a bundle image and not useful for bundles created with --generate-only. "+
			"One of: [docker, podman, none]")
```

Reply:
I think the original is more clear, especially considering that this command can be used with non-SDK-generated bundles.

Reply:
IMO we need to make clear that it is not valid for the bundle created with the flag `--generate-only` explicitly, however, I am ok with any rephrase to address that.